Apps/WebClient/src/Server/Controllers/CommentController.cs

```diff
@@ -139,11 +139,28 @@ public IActionResult Delete(string hdid, [FromBody] UserComment comment)
         /// <response code="401">The client must authenticate itself to get the requested response.</response>
         /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
         [HttpGet]
-        [Route("{hdid}")]
+        [Route("{hdid}/[controller]/Entry")]
         [Authorize(Policy = UserProfilePolicy.Read)]
         public IActionResult GetAllForEntry(string hdid, [FromQuery] string parentEntryId)
         {
-            RequestResult<IEnumerable<UserComment>> result = this.commentService.GetList(hdid, parentEntryId);
+            RequestResult<IEnumerable<UserComment>> result = this.commentService.GetEntryComments(hdid, parentEntryId);
+            return new JsonResult(result);
+        }
+
+        /// <summary>
+        /// Gets all comments for the authorized user.
+        /// </summary>
+        /// <param name="hdid">The user hdid.</param>
+        /// <returns>The list of comments wrapped in a request result.</returns>
+        /// <response code="200">Returns the list of comments.</response>
+        /// <response code="401">The client must authenticate itself to get the requested response.</response>
+        /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
+        [HttpGet]
+        [Route("{hdid}/[controller]")]
+        [Authorize(Policy = UserProfilePolicy.Read)]
+        public IActionResult GetAll(string hdid)
```

Comment:
Should we measure and update the unit tests as a part of this PR or create a new task for it as this new method will need to be tested?

Reply:
This is part of the merge from dev.